group_vars/all/globals.yml

```diff
@@ -36,3 +36,6 @@ postgresql_archive_command: >
   {% if wal_e_aws_instance_profile %}--aws-instance-profile{% endif %} wal-push %p
 
 postgres_master_host: "{{ deploy_env }}-postgres-master.{{ domain_name }}"
+
+elasticsearch_host_name: "{{ hosts_prefix }}-tsuru-elasticsearch"
+elasticsearch_url: "{{ deploy_env }}-elasticsearchapi.{{ domain_name }}"
```

Comment:
<server_type>_host is our usual nomenclature for that type of value, I would not like to change it just for elasticsearch.
This is also reason why I would not like to change _host to _ip... . It seems reasonable, but then I should rename all the variables in the proper way... (something _host_name -> something_server_name; something_host -> something_ip; and finally use something_host for host names).